PROTOCOL.md

```diff
@@ -326,6 +340,31 @@ The following is an example `remove` action.
 }
 ```
 
+### Add CDC File
+The `cdc` action is used to add a [file](#change-data-files) containing only the data that was changed as part of the transaction.
+
+The schema of the `cdc` action is as follows:
+
+Field Name | Data Type | Description
+-|-|-
+path| String | A relative path to a file from the root of the table or an absolute path to a file that should be removed from the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt), which needs to be decoded to get the file path.
+partitionValues| Map[String, String] | A map from partition column to value for this file. See also [Partition Value Serialization](#Partition-Value-Serialization)
+size| Long | The size of this file in bytes
+dataChange | Boolean | Should always be `false` for change data because it only mirrors the effective changes of the data files
```

Comment:
AddCDCFile doesn't have `dataChange`. it has `tags`. see https://github.com/delta-io/delta/blob/master/core/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala#L415-L420

Reply:
Resolved but it's interesting that the reference implementation still writes it as a consequence of `FileAction`

Reply:
oops. I'm wrong here. it has `dataChange`. sorry for that @nkarpov